docs/Ace.md

```diff
@@ -80,13 +79,22 @@ The definition of `check_consistency` and `clean_dir` as well as all other helpe
 For example, in running the following:
 
 ```
-python2 xfstestAdapter.py -b ../ace/base_xfstest.sh -t <J-LANG FILE> -p output/ -n 001 -f generic
+python3 xfstestAdapter.py -t <J-LANG FILE> -p output/ -n 001 -f generic
 ```
 
-would create `output/001` and `output/001.out` from the given J-lang file.  Note that the base file at `../ace/base_xfstest.sh` should be used for every use of the XFSTest adapter.
+would create `output/001` and `output/001.out` from the given J-lang file. 
+
+3. *XFSTEST-Concise* : Ace can also generate J-lang version two files (or J2-lang files) which contain multiple tests wrapped into a single file. The *XFSTest* adapter automatically detects what version of J-lang file is passed in, and decides whether to generate a single test or a condensed test. The adapter can be run with the same arguments as described above, with J-lang files replaced by J2-lang files. For example, running:
+ 
+```
```

Comment:
You can specify what type of test you want to create directly from Ace using the -t flag with the option *xfstest-concise*. It will create tests files 001, 001.out, 002, 002.out, etc. in code/tests/seq1 and  j2-lang1, j2-lang2, etc. in code/tests/seq1/j-lang-files, just like w/ Crashmonkey.

The flag is specified at the bottom of docs/Ace.md:

      * `-t` - The type of test to generate. Should be one of 'crashmonkey', 'xfstest', or 'xfstest-concise'. If unspecified, the adapter will default to 'crashmonkey'.